contrib/opencensus-ext-azure/opencensus/ext/azure/common/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.2.1'
+__version__ = '0.2.2'
```

Comment:
Micro version updates are usually small and backwards-compatible, this looks like at least a minor-level change to me. https://www.python.org/dev/peps/pep-0440/#semantic-versioning.